games-emulation/retroarch-assets/retroarch_assets-git20180129.recipe

```diff
@@ -0,0 +1,32 @@
+SUMMARY="Media assets for RetroArch"
+DESCRIPTION="RetroArch is a frontend for emulators, game engines and media \
+players. This package provides assets such as icons for the GUI."
+HOMEPAGE="https://libretro.com/"
+COPYRIGHT="2010-2018 The RetroArch Team"
+LICENSE="CC-BY-4.0"
+REVISION="1"
+srcGitRev="505ad6205e5fd8a155daad02ab39953a0b15b0a9"
+SOURCE_URI="https://github.com/libretro/retroarch-assets/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="de37772b00b755b80aafe5d4df0713c0fe6b0e5a79017d94e43f279e798aa85d"
+SOURCE_FILENAME="retroarch-assets-$portVersion.tar.gz"
+SOURCE_DIR="retroarch-assets-$srcGitRev"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	retroarch_assets = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+INSTALL()
+{
+	retroarchDataDir="$dataDir/retroarch/assets/"
+	mkdir -p "$retroarchDataDir"
+	cp -ar * "$retroarchDataDir"
+	cd "$retroarchDataDir"
+	rm -rf src 
```

Comment:
```diff
-	rm -rf src 
+	rm -rf src
```
(trailing whitspace at the end of the line)

Reply:
Whoops! Thanks. Fixed that and renamed recipe.